packages/i18n/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "@node-core/website-i18n",
+  "type": "module",
+  ".": {
+    "types": "./dist/index.d.mts",
+    "default": "./lib/index.mjs"
+  },
+  "main": "./lib/index.mjs",
+  "module": "./lib/index.mjs",
+  "types": "./dist/index.d.mts",
+  "files": [
+    "locales"
+  ],
+  "scripts": {
+    "build": "tsc --declaration --emitDeclarationOnly",
+    "lint:js": "eslint \"**/*.{js,mjs,ts}\" --cache --cache-strategy=content --cache-location=.eslintjscache"
```

Per #7032, ESLint's native caching isn't compatible with the plugins list this repo uses. Specifically, some `import` rules.

```suggestion
    "lint:js": "eslint \"**/*.{js,mjs,ts}\""
```

(here and any other `.eslintjscache` references, such as the `.gitignore`)

---

So, remove all caching rules?

---

Note that this was done to fasten cache within CI 🤔